src/js/pages/Index.js

```diff
@@ -20,11 +21,12 @@ import SidebarStore from "../stores/SidebarStore";
 
 function getSidebarState() {
   return {
-    isDocked: SidebarStore.get("isDocked"),
     isVisible: SidebarStore.get("isVisible")
   };
 }
 
+const mobileTreshold = 994;
```

Comment:
![xkcd](https://imgs.xkcd.com/comics/random_number.png)

Reply:
😆  This is the media query we use to target table/mobile. I will add a comment.